itou/templates/admin/job_applications/change_form.html

```diff
@@ -0,0 +1,32 @@
+{% extends 'admin/change_form.html' %}
+
+{% block submit_buttons_bottom %}
+
+    <div class="submit-row">
+        {% if original.state.is_new %}
+            <input class="danger with-confirm" type="submit" value="Passer à l'étude" name="_process">
+        {% endif %}
+        {% if original.is_in_acceptable_state %}
+            <input class="danger with-confirm" type="submit" value="Accepter" name="_accept">
+        {% endif %}
+        {% if original.state.is_accepted and original.can_be_cancelled %}
+            <input class="danger with-confirm" type="submit" value="Annuler" name="_cancel">
+        {% endif %}
+        {% if original.state.is_obsolete %}
+            <input class="danger with-confirm" type="submit" value="Remettre au statut nouveau" name="_reset">
+        {% endif %}
+    </div>
+
+    {{ block.super }}
+
+    <script nonce="{{ CSP_NONCE }}">
+        var buttons = document.getElementsByClassName("with-confirm");
```

Comment:
Pour suivre la logique utilisée ailleurs, et bien plus pratique pour les retrouver :).
```suggestion
        var buttons = document.getElementsByClassName("js-with-confirm");
```